app/src/main/java/com/example/myapplication/ThirdFragment.java

```diff
@@ -55,15 +96,130 @@ public void onClick(View view) {
+                    binding.switch3.setTrackTintList(ColorStateList.valueOf(Color.parseColor("#73C2F0")));
+                }
+                MainActivity.broke = binding.switch3.isChecked();
+            }
+
+        });
+        lightDark(v, MainActivity.darkMode);
+
         String[] typeContact = {"No contact", "Teammate to Teammate", "Teammate to opponent"};
         ArrayAdapter<String> contact = new ArrayAdapter<>(view.getContext(), android.R.layout.simple_spinner_dropdown_item, typeContact);
         binding.spinner.setAdapter(contact);
 
 
-        Editable numNotes = binding.numNotes.getText();
 
+        Switch switch1 = binding.switch1.findViewById(R.id.switch1);
+        switch1.isChecked();
+
+        Switch switch2 = binding.switch2.findViewById(R.id.switch2);
+        switch2.isChecked();
+
+        Switch switch3 = binding.switch3.findViewById(R.id.switch3);
+        switch3.isChecked();
+
+        EditText numNotes = binding.numNotes.findViewById(R.id.numNotes);
+        numNotes.getText();
+
+        EditText numNotesInAmp = binding.editTextNumber2.findViewById(R.id.editTextNumber2);
+        numNotesInAmp.getText();
+        }
+    @Override
+    public void onDestroyView() {
+        super.onDestroyView();
+
+        binding = null;
+
+    }
+    public void lightDark (ViewGroup v, boolean mode){
+        if (!mode){
+            for (int i = 0; i < v.getChildCount(); i ++){
+                View child = v.getChildAt(i);
+                if (!(child instanceof Button)) {
+                    child.setBackgroundColor(Color.parseColor("#FFFFFF"));
+                    if (child instanceof TextView) {
+                        TextView tx = (TextView) child;
+                        tx.setTextColor(Color.parseColor("#000000"));
+                    }
+                }
+                if (child instanceof Switch) {
+                    Switch tx = (Switch) child;
+                    tx.setTextColor(Color.parseColor("#000000"));
+                }
+                if (child instanceof ViewGroup){
+                    lightDark((ViewGroup)child, mode);
+                }
+            }
+        } else {
+            for (int i = 0; i < v.getChildCount(); i ++){
+                View child = v.getChildAt(i);
+                if (!(child instanceof Button)) {
+                    child.setBackgroundColor(Color.parseColor("#000000"));
+                    if (child instanceof TextView) {
+                        TextView tx = (TextView) child;
+                        tx.setTextColor(Color.parseColor("#FFFFFF"));
+                    }
+                }
+                if (child instanceof Switch) {
+                    Switch tx = (Switch) child;
+                    tx.setTextColor(Color.parseColor("#FFFFFF"));
+                }
+                if (child instanceof ViewGroup){
+                    lightDark((ViewGroup)child, mode);
+                }
+            }
```

Comment:
This could be simplified as 

        Color bgColor = mode ? Color.parseColor("#000000") : Color.parseColor("#FFFFFF");
        Color textColor = mode ? Color.parseColor("#FFFFFF") : Color.parseColor("#000000");
        for (int i = 0; i < v.getChildCount(); i ++){
            View child = v.getChildAt(i);
            if (!(child instanceof Button)) {
                child.setBackgroundColor(lightColor);
                if (child instanceof TextView) {
                    TextView tx = (TextView) child;
                    tx.setTextColor(textColor);
                }
            }
            if (child instanceof Switch) {
                Switch tx = (Switch) child;
                tx.setTextColor(textColor);
            }
            if (child instanceof ViewGroup){
                lightDark((ViewGroup)child, mode);
            }
        }

since the if else statement is basically copying itself except for small value changed, once you fix it can you also mention it to  @MthreeJohnnyK  so he can simplify the code on his pages as well?